datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx

```diff
@@ -120,68 +171,160 @@ export const EntityHeader = () => {
+                                <PlatformDivider />
+                                <ContainerIcon
+                                    style={{
+                                        color: ANTD_GRAY[9],
+                                    }}
+                                />
+                                <ContainerText>
+                                    {entityRegistry.getDisplayName(EntityType.Container, container)}
+                                </ContainerText>
+                            </Link>
+                        )}
+                        {entityCount && entityCount > 0 ? (
+                            <>
+                                <PlatformDivider />
+                                <EntityCountText>{entityCount.toLocaleString()} entities</EntityCountText>
+                            </>
+                        ) : null}
+                    </PlatformContent>
+                    <div style={{ display: 'flex', justifyContent: 'left', alignItems: 'center' }}>
+                        <Link to={entityPath}>
+                            <EntityTitle level={3}>{entityData?.name || ' '}</EntityTitle>
                         </Link>
-                    )}
-                    {entityCount && entityCount > 0 ? (
-                        <>
-                            <PlatformDivider />
-                            <EntityCountText>{entityCount.toLocaleString()} entities</EntityCountText>
-                        </>
-                    ) : null}
-                </PlatformContent>
-                <div style={{ display: 'flex', justifyContent: 'left', alignItems: 'center' }}>
-                    <Link to={entityPath}>
-                        <EntityTitle level={3}>{entityData?.name || ' '}</EntityTitle>
-                    </Link>
-                    {entityData?.health && (
-                        <EntityHealthStatus
-                            status={entityData?.health.status}
-                            message={entityData?.health?.message || undefined}
-                        />
-                    )}
-                </div>
-            </MainHeaderContent>
-            {hasExternalUrl && (
-                <ExternalLinkButton href={externalUrl} onClick={sendAnalytics}>
-                    View in {platformName}
-                </ExternalLinkButton>
-            )}
-            <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
-                <Button
-                    icon={copiedUrn ? <CheckOutlined /> : <CopyOutlined />}
-                    onClick={() => {
-                        navigator.clipboard.writeText(urn);
-                        setCopiedUrn(true);
-                    }}
-                />
-            </Tooltip>
-        </HeaderContainer>
+                        {entityData?.deprecation?.deprecated && entityData?.deprecation?.decommissionTime !== null && (
```

Comment:
Minor: Let's put this condition in a variable above^^ (line 227) that is called "isDeprecated" --> I think we can also remove the && `entityData?.deprecation?.decommissionTime !== null`

Reply:
`{entityData?.deprecation?.deprecated &&
                            (entityData?.deprecation?.note !== '' ||
                                entityData?.deprecation?.decommissionTime !== null) && (
                                <Popover
                                    overlayStyle={{ maxWidth: 240 }}
                                    placement="right"
                                    content={
                                        <>
                                            {entityData?.deprecation?.note !== '' && (
                                                <Typography.Text>{entityData?.deprecation?.note}</Typography.Text>
                                            )}
                                            {entityData?.deprecation?.decommissionTime !== null && (
                                                <Typography.Text type="secondary">
                                                    <Tooltip placement="right" title={decommissionTimeGMT}>
                                                        <LastEvaluatedAtLabel
                                                            style={{
                                                                borderTop: '1px solid #f0f0f0',
                                                                paddingTop: '5px',
                                                            }}
                                                        >
                                                            {decommissionTimeLocal}
                                                        </LastEvaluatedAtLabel>
                                                    </Tooltip>
                                                </Typography.Text>
                                            )}
                                        </>
                                    }
                                >
                                    <DeprecatedContainer>
                                        <InfoCircleOutlined />
                                        <DeprecatedText>Deprecated</DeprecatedText>
                                    </DeprecatedContainer>
                                </Popover>
                            )}
                        {entityData?.deprecation?.deprecated &&
                            entityData?.deprecation?.decommissionTime === null &&
                            entityData?.deprecation?.note === '' && (
                                <DeprecatedContainer>
                                    <InfoCircleOutlined />
                                    <DeprecatedText>Deprecated</DeprecatedText>
                                </DeprecatedContainer>
                            )}
`